python/equistore-core/equistore/core/labels.py

```diff
@@ -303,6 +303,144 @@ def range(name: str, end: int) -> "Labels":
             values=np.arange(end, dtype=np.int32).reshape(-1, 1),
         )
 
+    def append(self, name: str, values: np.ndarray) -> "Labels":
```

Comment:
If we add function to the Labels in equistore-core, we need to add the same ones in equistore-torch. 

The API is nicer if we add member functions, but that's more work to do. I don't have strong feelings between adding functions twice or keeping them in equistore-operations

Reply:
Yes, I will add them to `equistore-torch` if we agree on the API. I think the fit quite good here. I only haven't touched this to avoid changing lines twice...